libs/core/src/components/pds-tooltip/pds-tooltip.tsx

```diff
@@ -11,10 +9,22 @@ import {
 @Component({
   tag: 'pds-tooltip',
   styleUrls: ['pds-tooltip.scss'],
-  shadow: true,
+  shadow: false,
```

Comment:
Removing the shadow so the portal approach can be supported. This allows for inserting the tooltip before the closing body tag.